Makefile

```diff
@@ -21,7 +21,11 @@ lint-fix:
 
 .PHONY: test
 test:
-	pytest woodwork/
+	pytest woodwork/ --durations 10
```

Comment:
do we need the `--durations` flag here?

Reply:
I felt like this was a nice inclusion especially if we can transition in the future to using `make` commands in CI but definitely not needed for this PR, thanks!